EXAMPLES.md

```diff
@@ -148,9 +145,6 @@ end
 This spares one from having to figure out the correct escaping sequences for
 something like "echo(:cat, '...?...', '> /etc/sudoers.d/yolo')".
 
-**Note:** The `upload!()` method doesn't honor the values of `within()`, `as()`
-etc, this will be improved as the library matures, but we're not there yet.
```

Comment:
This (and the above) appeared to be inaccurate based on [my testing of #524](https://github.com/capistrano/sshkit/pull/524#pullrequestreview-1799265301).